spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteManifestsAction.java

```diff
@@ -466,6 +472,164 @@ public void testRewriteLargeManifestsPartitionedTable() throws IOException {
     assertThat(newManifests).hasSizeGreaterThanOrEqualTo(2);
   }
 
+  @TestTemplate
+  public void testRewriteManifestsPartitionedTableWithInvalidSortColumnsThowsException()
```

Comment:
These are very through tests

Reply:
Too detailed/fragile? I always struggle to come up with simple unit tests for things that involve contextual data layout.

Reply:
Sorry, I was complementing I think you are covering this well